flair/nn/model.py

```diff
@@ -582,42 +613,19 @@ def multi_label_threshold(self, x):  # setter method
         else:
             self._multi_label_threshold = {"default": x}
 
-    def forward_loss(self, sentences: Union[List[DT], DT]) -> Tuple[torch.Tensor, int]:
-
-        # make a forward pass to produce embedded data points and labels
-        embedded_data_points, labels = self.forward_pass(sentences)  # type: ignore
-
-        # no loss can be calculated if there are no labels
-        if not any(labels):
```

Comment:
This check is currently important for models that require candidates like the EntityLinker or RelationExtractor.